### PR TITLE
Merge and refine UI bundles changes

### LINKED ISSUE #1767: Outdated migration guide for 15898
# Problem

The migration guide for [Migrate UI bundles to required components](https://github.com/bevyengine/bevy/pull/15898) is outdated, because it tells me to use Style, but the migration guide for [Merge Style properties into Node. Use ComputedNode for computed properties.](https://github.com/bevyengine/bevy/pull/15975) tells me to abolish Style completely.

# Solution

Update the migration guide for 15898 to contain the full migration.

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fixes #1767.

### CODE REVIEW

**Review by richchurcher — APPROVED** [review]

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 1]
```diff
@@ -19,9 +68,51 @@
 
 - Replace all uses of `ButtonBundle` with `Button`. e.g.
 
-```diff
-                     .spawn((
--                        ButtonBundle {
+```diffMove any fields set on `Style` into `Node` and replace all `Style` component usage with `Node`.
```
Comment (IQuick143) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:71]: This diff seems to be cut in the middle, and also is missing a newline.

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 2]
```diff
@@ -19,9 +68,51 @@
...
+After:
+
+```rust
+fn system(computed_nodes: Query<&ComputedNode>) {
+    for computed_node in &computed_nodes {
+        let computed_size = computed_node.size();
+    }
+}
+```
+      ButtonBundle {
```
Comment (IQuick143) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:115]: I think it's supposed to continue here.

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 3]
```diff
@@ -19,9 +68,51 @@
...
+```
+
+After:
+
+```rust
+fn system(computed_nodes: Query<&ComputedNode>) {
+    for computed_node in &computed_nodes {
+        let computed_size = computed_node.size();
+    }
+}
```
Comment (IQuick143) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:113]: This section is weirdly repeated, is it a Copy-Paste error?

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 4]
```diff
@@ -1,71 +1,86 @@
...
+Style {
+    width:  Val::Px(100.),
+    ..default()
+}
+```
+
+After:
+
+```rust
+Style {
```
Comment (benfrankel) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:49]:
```suggestion
Node {
```

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 5]
```diff
@@ -1,71 +1,86 @@
...
--    });
-+    ));
+```rust
+     commands
+        .spawn(Node {
+                 width: Val::Percent(100.),
+                 align_items: AlignItems::Center,
+                 justify_content: JustifyContent::Center,
+                 ..default()
+             });
```
Comment (benfrankel) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:85]:
```suggestion
            width: Val::Percent(100.),
            align_items: AlignItems::Center,
            justify_content: JustifyContent::Center,
            ..default()
        });
```

**Review by benfrankel — CHANGES_REQUESTED** [review]

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 6]
```diff
@@ -1,71 +1,86 @@
...
+
+## `Style` -> `Node`
+
+All of the values of `Style` now live on `Node`. This is a find-and-replace rename.
+
+Before:
+
+```rust
+Style {
+    width:  Val::Px(100.),
```
Comment (benfrankel) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:41]:
```suggestion
    width: Val::Px(100.),
```

File: release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md [hunk 7]
```diff
@@ -1,71 +1,86 @@
...
+    ..default()
+}
+```
+
+After:
+
+```rust
+Node {
+    width:  Val::Px(100.),
```
Comment (benfrankel) [inline release-content/0.15/migration-guides/15898_Migrate_UI_bundles_to_required_components.md:50]:
```suggestion
    width: Val::Px(100.),
```

**Review by benfrankel — APPROVED** [review]

**Pull request merged**